### PR TITLE
Use Makefile as source of truth for commands

### PR DESCRIPTION
**Pull request by mattbennett**
This will simplify a number of the changes to ``tox.ini`` in #250. Raised as a separate change for visibility and to slim down that PR.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -25,25 +25,18 @@ deps =
 
     latest: eventlet  # need something
 
+whitelist_externals = make
+
 commands =
     pip install --editable .[dev]
-    flake8 nameko test
-    pylint --rcfile=pylintrc nameko -E
```
Comment (davidszotten) [inline tox.ini]: we ok without specifying the rc file?

File: tox.ini [hunk 2]
```diff
@@ -25,25 +25,18 @@ deps =
...
-    coverage run --concurrency=eventlet --source nameko -m \
-        pytest test {posargs:--timeout=30}
-    coverage report --show-missing --fail-under=100
```
Comment (davidszotten) [inline tox.ini]: show-missing is handy for debugging failed builds on travis

**Comment by davidszotten** [comment]
i'm sure there was some reason for the way we structured this, but can't recall it, so happy to try this way around

**Comment by mattbennett** [comment]
If I recall correctly we wanted to embrace tox but left the Makefile for convenience. This way we get both with less repetition, and you can dev locally without tox if you want.

Dropping ``show-missing`` and the ``rcfile`` was accidental, I'll add them back again.

**Comment by davidszotten** [comment]
:+1:

**Comment by mattbennett** [comment]
Superseded by https://github.com/onefinestay/nameko/pull/266

**Pull request closed without merging**